.codeclimate.yml

```diff
@@ -1,15 +1,167 @@
-engines:
-  golint:
+# This is our default .CodeClimate.yml, broken out by language. Uncomment the
+# sections at the bottom that apply to your project. ACTION comments indicate
+# places where config might need to be tweaked.
+
+version: "2"
+
+plugins:
+
+# ---------------
+# Cross-language plugins. Should always be on.
+
+  duplication: # Looks for similar and identical code blocks
     enabled: true
-    exclude_patterns:
-    - "**/"                # exclude all
-    - "!./cmd/"            # unexclude just the ones we want to lint
-    - "!./pkg/"
-    - "!./templates/"
+    config:
+      languages:
+        go:
+        java:
+        javascript:
+        php:
+        python:
+          python_version: 3  # ACTION Comment this out if using Python 2
+        ruby:
+        swift:
+        typescript:
```

Comment:
This repo will never contain some of these so they should be removed:
- java
- php
- swift

Reply:
We're not customizing the engines per repo - the goal is one master file everywhere. All of these engines add minimal time to the scan and cause no issues to run.